README.md

```diff
@@ -50,6 +50,9 @@ For incremental builds, the `rush build` command can be used to only build packa
 4. Locally commit changes: `git commit` (or use the Visual Studio Code user interface)
 5. Repeat steps 1-4 until ready to push changes
 6. Check for API signature changes: `rush extract-api`. This will update the signature files, located in `common/api`.
+    - Be sure that your branch is up to date with `master` (i.e. `git merge origin/master`)
```

Comment:
Up to date with target branch -- not necessarily master.

Reply:
Updated as per suggestion